src/components/LoginPop.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useContext, useEffect } from "react"
```

Comment:
This login modal is a nice touch, but it deletes any text already in the comment box.